samples/FunctionalDependencyModule.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace Botwin.Samples
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Botwin.Response;
+
+    public class FunctionalDependencyModule : BotwinModule
+    {
+        public FunctionalDependencyModule()
+        {
+            this.Get("/functional", async (req, res, routeData) =>
+            {
+                var handler = Composition.ComposeFunctionalHandler();
```

Comment:
How do we replace this in an acceptance in mem http pipeline test so we remove live sql connections